packages/app-store/routing-forms/pages/form-edit/[...appPages].tsx

```diff
@@ -174,6 +183,21 @@ function Field({
     updateSelectText(newList);
   }
 
+  const handlePaste = (event: ClipboardEvent, index: number) => {
+    const paste = event.clipboardData.getData("text");
+    const formattedValues = paste
+      .split(/[\n,;]+/)
```

Comment:
Do we need to split on , and ; ?  I think we should just split on new line and extend this feature if someone really need to have commas or semicolon support.

Reason being someone might find it suitable to provide a literal comma in one of their options.

Also, let's move this regex out as a constant in this module.